src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -2776,6 +2776,9 @@ in src-script.js and main.js
 {
 	margin-bottom: 0.75em;
 }
+.negative-marker {
+	display: none;
+}
```

Comment:
Is there any meaningful difference between an inline `span` with no content and the same span but hidden? if not, then I don't think this is needed.

Reply:
It makes the rendering engine completely discard the element. So close to no difference. But still better to have.